src/components/customRoute/customRoute.js

```diff
@@ -16,7 +16,7 @@ const CustomRoute = ({
   Piwik.tracking(rest.history, settings);
 
   return ((isPrivate && isAuthenticated) || !isPrivate ?
-    <main className={isPrivate ? offlineStyle.disableWhenOffline : null}>
+    <main className={`${isPrivate ? offlineStyle.disableWhenOffline : null} ${offlineStyle.setHight} offlineWrapper`}>
```

Comment:
```suggestion
    <main className={`${isPrivate ? offlineStyle.disableWhenOffline :''} ${offlineStyle.setHight} offlineWrapper`}>
```
Avoid null on template literals